tests/utils/common-utils.mjs

```diff
@@ -0,0 +1,354 @@
+import fs from 'fs-extra';
+import os from 'node:os';
+
+export class Constants {
+  static get cliPromptMsDelay() {
+    return 500;
+  }
+  static get minaGraphQlPort() {
+    return 8080;
+  }
```

Comment:
@mitschabaude but in your case such a properties won't be really a constants. Anyone can set new values anytime, while with static getters even if runtime won't throw you upon new value assignment, the actual value won't be changed and it will return.
Sure, for tests we probably should aim for simplicity and readability first, yet still it might be source of flakiness.

Reply:
Will leave it as it was for now if you don't mind.

Reply:
Or I'll just use `Object.freeze`.

Reply:
Oh if you want to ensure immutability it makes sense (I don't generally worry about that I guess)